---
/**:
  ros__parameters:
    # The client_id to be set in smart_access_config.json, must be a unique integer.
      master_client_id: 1000
      # This sets the dev_id in the hw_inventory.json
      hw_dev_id: 2
      # This sets the port in the hw_inventory.json
      hw_port: 55555
      # This sets the iface_name in the hw_inventory.json
      hw_iface_name: "eth0"
      # An array of sensors to subscribe to.
      sensors:
          # As many as 10 sensors all named as "sensor_<number>" in increasing order of numbers,
          # e.g., sensor_0, sensor_1, etc. The list must start with sensor_0.
          sensor_0:
              # The model of the connected sensor. Available models umrr11, umrr9d, umrr96,
              # umrr9f_v1_1_1, umrr9f_v2_0_0
              model: "umrr9d_v1_2_1"
              # The client_id of the sensor/source, must be a unique integer.
              id: 100
              # The ip address of the sensor or of the source acting as a sensor.
              ip: "192.168.11.11"
              # The port to be used.
              port: 55555
              # The frame_id to be set to the published messages.
              frame_id: "umrr"
              # Specify the history size.
              history_size: 10
          #sensor_1:
          #    # The model of the connected sensor.
          #    model: "umrr96"
          #    # The client_id of the sensor/source, must be a unique integer.
          #    id: 200
          #    # The ip address of the sensor or of the source acting as a sensor.
          #    ip: "172.22.10.102"
          #    # The port to be used.
          #    port: 55555
          #    # The frame_id to be set to the published messages.
          #    frame_id: "umrr"
          #    # Specify the history size.
          #    history_size: 10
          #sensor_2:
          #    # The model of the connected sensor.
          #    model: "umrr9f_v1_1_1"
          #    # The client_id of the sensor/source, must be a unique integer.
          #    id: 300
          #    # The ip address of the sensor or of the source acting as a sensor.
          #    ip: "172.22.10.103"
          #    # The port to be used.
          #    port: 55555
          #    # The frame_id to be set to the published messages.
          #    frame_id: "umrr"
          #    # Specify the history size.
          #    history_size: 10
          #sensor_3:
          #    # The model of the connected sensor.
          #    model: "umrr9d"
          #    # The client_id of the sensor/source, must be a unique integer.
          #    id: 400
          #    # The ip address of the sensor or of the source acting as a sensor.
          #    ip: "172.22.10.104"
          #    # The port to be used.
          #    port: 55555
          #    # The frame_id to be set to the published messages.
          #    frame_id: "umrr"
          #    # Specify the history size.
          #    history_size: 10
